README.md

```diff
@@ -3,12 +3,35 @@
 # Xlang
 
 ## Overview
-Xlang is a system that enables a component written in any supported language to be callable from any other supported language using object-oriented programming constructs.  
+The xlang project enables developers to take existing shared libraries,
+implemented in one programming language,
+and make that library's APIs available to client code using a different programming language.
+Thus the name "xlang," for cross-language.
+
+Additionally, all of the xlang tooling will be platform-agnostic therefore will work on your favorite operating system.
```

Comment:
The wording here is a bit misleading. The tooling will be available on all platforms, but it's not always identical. At the least it's recompiled. "will be available on a variety of platforms and written using portable code wherever possible to make it easy to port to your favorite operating system"